libraries/AP_BattMonitor/AP_BattMonitor_Params.cpp

```diff
@@ -13,7 +13,7 @@ const AP_Param::GroupInfo AP_BattMonitor_Params::var_info[] = {
     // @Param: MONITOR
     // @DisplayName: Battery monitoring
     // @Description: Controls enabling monitoring of the battery's voltage and current
-    // @Values: 0:Disabled,3:Analog Voltage Only,4:Analog Voltage and Current,5:Solo,6:Bebop,7:SMBus-Generic,8:DroneCAN-BatteryInfo,9:ESC,10:SumOfFollowing,11:FuelFlow,12:FuelLevelPWM,13:SMBUS-SUI3,14:SMBUS-SUI6,15:NeoDesign,16:SMBus-Maxell,17:Generator-Elec,18:Generator-Fuel,19:Rotoye,20:MPPT,21:INA2XX,22:LTC2946,23:Torqeedo
```

Comment:
I'm not sure "Monitors" adds clarity here.  I think "Sum of All Monitors" or "Sum of All" or something like that might be better.

Reply:
it is not ALL unless you set ALL the mask bits....can be any number of monitors

Reply:
Maybe we should just make it simply "Sum" then?

It's no big deal but the "of Monitors" part doesn't add value because what else could you be summing besides battery monitors?  I guess you could make it "Sum of Mask" which people may not initially understand but it might push them towards the BATTx_SUM_MASK parameter..

Reply:
how about Sum of Selected Monitors?

Reply:
sure that sounds better, txs